frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-details.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { getResource } from 'kubevirt-web-ui-components';
+
+import {
+  Firehose,
+  StatusBox,
+  ScrollToTopOnMount,
+  SectionHeading,
+} from '@console/internal/components/utils';
+import { TemplateKind } from '@console/internal/module/k8s';
+import { TemplateModel } from '@console/internal/models';
+
+import { VMTemplateResourceSummary, VMTemplateDetailsList } from './vm-template-resource';
+import { DataVolumeModel } from '../../models';
+
+export const VMTemplateDetailsFirehose: React.FC<VMTemplateDetailsFirehoseProps> = ({
+  obj: template,
+}) => {
+  const { name, namespace } = template.metadata;
+
+  const vmtRes = getResource(TemplateModel, {
```

Comment:
can you please remove the unnecessary template loading as discussed?

Reply:
sure :+1:

Reply:
Does this mean `vmtRes` should be removed?

Reply:
yes :-)

Reply:
@suomiy I am trying to remove the  unnecessary template loading. but I get stuck on the firehose decorator.
https://github.com/openshift/console/blob/master/frontend/public/components/utils/firehose.jsx#L190
because `DataVolume` is not declared on my system I get `loaded=false` always, and the page is not rendered:
https://github.com/openshift/console/blob/master/frontend/public/components/utils/firehose.jsx#L164

what am I missing ?